homework/style.css

```diff
@@ -1,3 +1,78 @@
+header {
+    background-color: rgb(116, 56, 13); 
+    color: white; 
+    padding: 1rem; 
+}
+
+body {
+    background-color:rgb(197, 197, 197);
+    color: black;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    font-size: 1rem;
+    padding-right: 1px;
+}
+
 .alert-error {
     color: red;
+}
+
+
+select {
+    font-size: .9rem;
+    padding: 4px 25px;
+    margin-left: 10px;
+    border-radius: 20px;
+}
+
+#container {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+
+    margin: 1 rem;
+}
+
+#forRepoBlock {
+    color: rgb(43, 0, 43);
+    font-size: .9rem;
+    padding: 4px 4px;
+    margin-left: 2px;
+    margin-top: 1rem;
+    width: 60%;
+
+    border-style: bold;
+    border-radius: 0%;
+    padding: 10px;
+    background:rgba(230, 178, 210, 0.5);
+    opacity: 1;
+
+    box-shadow: 10px 5px 5px grey;
+}
+#forContributorsBlock {
+    color: rgb(48, 0, 20);
+    font-size: .9rem;
+    padding: 4px 4px;
+    margin-left: 2px;
+
+    display: grid;
+    grid-template-columns: 33.3333% 33.3333% 33.3333%;
```

Comment:
It's probably better to use flex-sizing here: `grid-template-columns: 1fr 1fr 1fr;` (three columns with the same flex factor = the same width)

Reply:
fixed, thank you :)